sdk-node/src/workflows/workflow.ts

```diff
@@ -291,19 +307,6 @@ export class Workflow<TInput extends WorkflowInput, name extends string> {
       executionId,
     });
 
-    const l1m = new L1M({
```

Comment:
We will probably want to instantiate one on ctx also. Otherwise we can't pass x-workflow-execution-id which allows us to track model calls on the workflow (And render them in the UI).

Reply:
Oh, this is an oversight.